test/tests/ver-1c/comparison_ver-1c.py

```diff
@@ -3,41 +3,99 @@
 import numpy as np
 from matplotlib import gridspec
 import pandas as pd
-from    scipy import special
+from scipy.special import erf
+from numpy import sqrt
 
-#========= Comparison of concentration as a function of time ===================
+tmap_sol = pd.read_csv("ver-1c_csv.csv")
```

Comment:
```suggestion
tmap_sol = pd.read_csv("./gold/ver-1c_csv.csv")
```
We should be using the gold file for this.